notebooks/239-image-bind/239-image-bind-quantize-nncf.ipynb

```diff
@@ -0,0 +1,1570 @@
+{
```

Comment:
<p>can you please also provide links on documentation page or/and nncf repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>